CONTRIBUTING.md

```diff
@@ -116,8 +169,10 @@ When appropriate, reference issues (via `#` plus number) .
 ### Python style guide
 
 For python code style, we use [black](https://github.com/psf/black).
+For markdown style, we use [prettier](https://prettier.io/).
```

Comment:
Would this be better situated in the `### Documentation style guide` section below?

Reply:
Done